doc/nrf/device_guides/working_with_nrf/nrf70/nrf7002eb_54h20dk_gs.rst

```diff
@@ -0,0 +1,107 @@
+
+Overview
+********
+
+The nRF7002 EB is a shield that can be used with the nRF54H20 DK to add support for Wi-Fi.
+
+Limitations
+===========
+
+The Wi-Fi support is experimental and has the following limitations:
+
+* It only supports STA mode.
+* It only allows Wi-Fi open security.
+* It is only suitable for low throughput applications.
+
+Pin mapping for nRF54H20 DK
+===========================
+
+The pin mapping for the nRF54H20 DK is as follows:
+
++------------------+-----------------------+-----------------------------------------------+
+| nRF700x pin name | nRF54H20 DK pins       |  Function                                     |
++==================+=======================+===============================================+
+| CLK              | P1.01                 | SPI Clock                                     |
++------------------+-----------------------+-----------------------------------------------+
+| CS               | P1.04                 | SPI Chip Select                               |
++------------------+-----------------------+-----------------------------------------------+
+| MOSI             | P1.05                 | SPI MOSI                                      |
++------------------+-----------------------+-----------------------------------------------+
+| MISO             | P1.06                 | SPI MISO                                      |
++------------------+-----------------------+-----------------------------------------------+
+| BUCKEN           | P1.00                 | Enable Buck regulator                         |
++------------------+-----------------------+-----------------------------------------------+
+| IOVDDEN          | P1.00                 | Enable IOVDD regulator                        |
++------------------+-----------------------+-----------------------------------------------+
+| IRQ              | P1.02                 | Interrupt from nRF7002                        |
++------------------+-----------------------+-----------------------------------------------+
+| GRANT            | P1.03                 | Coexistence grant from nRF7002                |
++------------------+-----------------------+-----------------------------------------------+
+| REQ              | P1.08                 | Coexistence request to nRF7002                |
++------------------+-----------------------+-----------------------------------------------+
+| STATUS           | P1.09                 | Coexistence status from nRF7002               |
++------------------+-----------------------+-----------------------------------------------+
+
+
+Minimum requirements
+********************
+
+Make sure you have all the required hardware and that your computer and mobile device both have one of the supported operating systems.
+
+Hardware
+========
+
+* nRF54H20 DK
+* nRF7002 EB
+* A suitable battery
+* Micro-USB 2.0 cable
+* USB-C charger
+* Jumper wires
```

Comment:
```suggestion
Hardware
========

You must have the following hardware: 

* nRF54H20 DK
* nRF7002 EB
* A suitable battery
* Micro-USB 2.0 cable
* USB-C charger
* Jumper wires
```

Reply:
Regarding Jumper wires, how many (minimum) are necessary? That should be put in, something like,

``Jumper wires - one for each <text here>``, or whatever the minimum is

Reply:
I suggest not to include any text after Hardware to avoid redundancy, as its already there after Minimum Requirements sub-heading. See https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gsg_guides.html.

I would suggest to keep the Jumper wires generic for now.
@krish2718: Will yo add the details after RC1?

Reply:
Yes, that's why I suggested not to include "You must have the following hardware: " text after **Hardware**.